content/reference-rippled.md

```diff
@@ -6396,7 +6488,7 @@ The response follows the [standard format](#response-formatting), with a success
 | tx_blob | String | Binary representation of the fully-qualified, signed transaction, as hex |
 | tx_json | Object | JSON specification of the [complete transaction](reference-transaction-format.html) as signed, including any fields that were automatically filled in |
 
-__*Caution:*__ If this command results in an error messages, the message can contain the account secret from the request. Make sure that these errors are not visible to others, including:
+**Caution:** If this command results in an error messages, the message can contain the secret key from the request. Make sure that these errors are not visible to others, including:
```

Comment:
The last sentence doesn't quite flow into the list. I think it would work a little better if you omitted "including."